libvast/vast/table_slice_row.hpp

```diff
@@ -62,7 +63,7 @@ class table_slice_row {
   }
 
 private:
-  table_slice_ptr slice_ = nullptr;
+  table_slice slice_ = {};
```

Comment:
The explicit assignment should not be necessary.

Reply:
It is for the default constructor.